src/content/zel-vue/getting-started.mdx

```diff
@@ -26,3 +26,58 @@ A Vue Component Library using Zeppelin Element Library
 ```
 yarn add @zlab-de/zel-vue
 ```
+
+## How to use our components
+
+You must bring in **zeppelin-element-library.css** to apply the styles from Zeppelin Element Library. It only needs to be brought it once at the root of your application.
+
+
+```js
+<template>
+  <div id="app" class="theme-zeppelin">
+    <zel-vue-button variant="primary">Primary Button</zel-vue-button>
+  </div>
+</template>
+
+<script>
+import "@zlab-de/zel-vue/zeppelin-element-library.css"
+import { ZelVueButton } from "@zlab-de/zel-vue";
+
+export default {
+  name: "App",
+  components: {
+    ZelVueButton
+  }
+};
+</script>
+```
+
+## Tip: Rename on import
+
+```
+<template>
+  <div id="app" class="theme-zeppelin">
+    <zButton variant="primary">Primary Button</zButton>
+  </div>
+</template>
+
+<script>
+import "@zlab-de/zel-vue/zeppelin-element-library.css";
+import { ZelVueButton } from "@zlab-de/zel-vue";
+
+export default {
+  name: "App",
+  components: {
+    ZButton: ZelVueButton
+  }
+};
+</script>
+```
+
+## theme.json from Zeppelin Element Library
+
+This repository also have a **theme.json** file available for you to help with styling. This file contains the following infomation (see the guidelines section for more information); colors, spacing, and typography specifications. Its intended use if for users using a CSS-in-JS solution to be able to call colors and spacing easier.
```

Comment:
some typos. Should be:
This repository also has a **theme.json** file available for you to help with styling. This file contains the following information (see the guidelines section for more information): colors, spacing and typography specifications. Its intended use is for users using a CSS-in-JS solution to be able to call colors and spacing easier.